linode/ipv6range/framework_datasource.go

```diff
@@ -29,6 +31,8 @@ func (d *DataSource) Read(
 	req datasource.ReadRequest,
 	resp *datasource.ReadResponse,
 ) {
+	tflog.Debug(ctx, "Read data.linode_ipv6range")
```

Comment:
nit: Probably better to logging the data source's name as `data.linode_ipv6_range`? Because it will align with its name from config.